utils/python/tests/scripts_regression_tests

```diff
@@ -12,8 +12,8 @@ from  CIME.XML.machines import Machines
 from  CIME.XML.files import Files
 from  CIME.case import Case
 
-SCRIPT_DIR = CIME.utils.get_acme_scripts_root()
-
+SCRIPT_DIR = CIME.utils.get_scripts_root()
+TOOLS_DIR = os.path.join(SCRIPT_DIR,"Tools")
```

Comment:
I think that the issue is to keep the scripts directory minimal with just the stuff that is going to be used outside of a case, stuff to be used in a case is in the scripts/Tools directory.